tests/Benchmarks/System.Buffers/Reader_Binary.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using System.Buffers.Reader;
+
+namespace System.Buffers.Benchmarks
+{
+    public class Reader_Binary
+    {
+        static byte[] s_array;
```

Comment:
nit: add access modifiers explicitly

Reply:
Yeah, sorry, copying an existing test. I'll fix that one too.